Essentials/src/main/java/com/earth2me/essentials/Settings.java

```diff
@@ -343,28 +342,30 @@ public BigDecimal getCommandCost(final IEssentialsCommand cmd) {
         return getCommandCost(cmd.getName());
     }
 
-    private ConfigurationSection _getCommandCosts() {
-        if (config.isConfigurationSection("command-costs")) {
-            final ConfigurationSection section = config.getConfigurationSection("command-costs");
-            final ConfigurationSection newSection = new MemoryConfiguration();
-            for (final String command : section.getKeys(false)) {
+    private CommentedConfigurationNode _getCommandCosts() {
+        final Map<String, CommentedConfigurationNode> section = ConfigurateUtil.getMap(config.getSection("command-costs"));
+        if (!section.isEmpty()) {
+            final CommentedConfigurationNode newSection = config.newSection();
```

Comment:
this seems like the wrong approach -- you are deserializing to a node, rather than a native object.